app/scripts/modules/netflix/fastProperties/domain/propertyCommnadType.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum PropertyCommandType {
```

Comment:
These are only for front end logic.  The enums are good as ordinal values for me in this case.